components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -455,11 +455,14 @@ interface PageHandler {
   // Used by the brave://wallet page to open up the approval panel
   ShowApprovePanelUI();
 
-  // Android only - Used by the brave://wallet page to open up the backup UI.
+  // Mobile only - Used by the brave://wallet page to open up the backup UI.
   ShowWalletBackupUI();
 
-  // Android only - Used by the brave://wallet page to open up the unlock UI.
+  // Mobile only - Used by the brave://wallet page to open up the unlock UI.
   UnlockWalletUI();
+
+  // iOS only - Used by Wallet WebUI when the user tap to open onboarding flow.
```

Comment:
`[EnableIf=is_ios]` maybe?

Reply:
Though not sure if its possible to implement this correctly on the TypeScript side if you only compile it with iOS

Reply:
didnt know we can do [EnableIf=is_ios]
yeah i see some other places use it too but all of them are used in backend and can be partially compile using a build flag. I don't think TypeScript can do that?
Other ideas are separating onboarding-welcome.tsx into iOS and non-iOS. Or a dedicated mojom interface for iOS? but both of them feel like a overkill